src/ToDo/Presentation/Dialogs/ExpirationDateViewModel.cs

```diff
@@ -1,6 +1,41 @@
 namespace ToDo.Presentation.Dialogs;
 
+[ReactiveBindable]
 public partial class ExpirationDateViewModel
 {
-    
+	private readonly INavigator _navigator;
+
+	public DateTime DueDate
+	{
+		get;
+		set;//TODO: the idea is return to TaskViewModel with the DueDate value / call BackToTaskMethod
```

Comment:
Do we have an issue linked for this TODO ?

Reply:
Not yet..

Reply:
Actually I'm trying to do that, and the reason why I created this draft PR is to receive suggestion/advice how to approach that

Reply:
I think we should just be using a simple button with a not_mobile:DatePickerFlyout or mobile:NativeDatePickerFlyout for flyout, and bind a starting value.
As for returning a value, you can hook into DatePickerFlyout.DatePicked to handle that. Best would be to create a behavior to wrap the event into a command. OR... simply two-way bind the date, and detect the changes in values.

Reply:
For now the the easiest solution will indeed to use a `DatePicker` and data bind two-way the value to a `DueDate` property. However, you should use a state for that, and then use the `State.Execute()` to automatically navigate back when a date has been chosen (cf. suggestions below).